index.html

```diff
@@ -433,10 +442,10 @@ <h4>Transformation (eddsa-2022)</h4>
 identifier (<var>cryptosuite</var>). A <em>transformed data document</em> is
 produced as output. Whenever this algorithm encodes strings, it MUST use UTF-8
 encoding.
-          </p>
+        </p>
```

Comment:
Looks like these indent changes are reverting the fixes from https://github.com/w3c/vc-di-eddsa/pull/34.  Can you please undo all of them as appropriate?

Reply:
Any easy way to do that?